pagination/paginator.go

```diff
@@ -0,0 +1,118 @@
+package pagination
+
+import (
+	"fmt"
+	"math"
+)
+
+const defaultPageNumber = 1
+const defaultItemsPerPage = 10
+
+// PaginatorInterface defines what makes a paginator
+type PaginatorInterface interface {
```

Comment:
thanks, I added "Interface" as it conflicted with the actual Paginator, I didn't end up using this in the end though so have removed.